package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name":                   "taskcluster-secrets",
+  "version":                "1.0.0",
+  "private":                true,
+  "scripts": {
+    "test":                 "./test/runtests.sh"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/taskcluster/taskcluster-secrets.git"
+  },
+  "dependencies":  {
+    "debug":                 "^2.1.3",
+    "lodash":                "^3.6.0",
+    "promise":               "6.1.0",
+    "taskcluster-base":      "0.7.25",
+    "babel":                 "^5.0.9",
+    "taskcluster-client":    "^0.22.0",
+    "kexec":                 "^1.2.0",
+    "octokat":               "0.4.11",
+    "yaml-js":               "0.1.1",
+    "slugid":                "^1.1.0",
+    "json-parameterization": "0.1.0"
```

Comment:
Eeep, I stole the package from another project and did not get around to trimming the dependencies. I'll add that.